Makefile

```diff
@@ -100,23 +100,24 @@ ifdef TAG
 ENV_VARS+=-e TAG="$(TAG)"
 endif
 
-RUN = $(CONTAINER_CLI) run -t -i --sig-proxy=true -u $(UID):$(GID) --rm \
```

Comment:
This is no need to start the container in interactive and terminal mode because this is a automation process. And it will cause some trouble if the caller process didn't get assigned to a tty, so I just removed them.